test/lint_test.rb

```diff
@@ -0,0 +1,59 @@
+require "bundler/setup"
+require "safe_yaml"
+require "minitest/autorun"
+require "net/http"
+
+describe "lint test" do
+  BANNED_WORDS = Regexp.new('\b(' + [
+    "GitHub App",     # They're all GitHub Apps
+    "bot",            # TODO: link to docs about why bot is not preferred
+    "automatically"   # They're all automatic
```

Comment:
Maybe add "probot" or "probot app" here, since they're all probot apps if they're on the site.